phpunit.xml.dist

```diff
@@ -15,7 +15,16 @@
 
     <filter>
         <whitelist processUncoveredFilesFromWhitelist="true">
-            <directory suffix=".php">src</directory>
+            <directory>./</directory>
+            <exclude>
+                <directory>./Resources</directory>
+                <directory>./Tests</directory>
+                <directory>./vendor</directory>
+            </exclude>
         </whitelist>
     </filter>
+
+    <logging>
+        <log type="coverage-clover" target="build/phpunit/clover.xml"/>
+    </logging>
```

Comment:
to be removed as it is already done by make target

Reply:
We should probably keep it here and remove it from Makefile, and so unify the whole config here